calabash-cucumber/lib/calabash-cucumber/store/preferences.rb

```diff
@@ -13,14 +14,80 @@ def initialize
         @path = File.join(dot_dir, "preferences", "preferences.json")
       end
 
+      # !@visibility private
+      def usage_tracking
+        preferences = read
+
+        unless valid_user_tracking_value?(preferences[:usage_tracking])
+          log_defaults_reset
+          preferences[:usage_tracking] = defaults[:usage_tracking]
+          write(preferences)
```

Comment:
I don't think we should write anything when calling this method, can't we just have defaults if unset?